.pre-commit-config.yaml

```diff
@@ -1,7 +1,7 @@
-files: 'momepy\/'
+files: 'docs\/|benchmarks\/|momepy\/'
```

Comment:
```suggestion
```

We can probably remove this altogether at this point, no?

Reply:
Sure. I'll give it a shot.